openedx_events/learning/signals.py

```diff
@@ -7,3 +7,147 @@
 They also must comply with the payload definition specified in
 docs/decisions/0003-events-payload.rst
 """
+
+from openedx_events.learning.data import CertificateData, CohortData, CourseEnrollmentData, UserData
+from openedx_events.tooling import OpenEdxPublicSignal
+
+# ..event_type: org.openedx.learning.student.registration.completed.v1
+# ..event_name: STUDENT_REGISTRATION_COMPLETED
+# ..event_version: V1
+# ..event_implementation: OpenEdxPublicSignal(DjangoSignal)
+# ..event_description: emitted when the user registration process in the LMS
+# is completed.
```

Comment:
Since we allow for 120 chars, my preference would be to make this more readable by not wrapping unless we get into long descriptions.

Reply:
I'll update the repo tox configurations, so we also support 120 chars

Reply:
Hey! I was about to change `max-doc-length = 79` for `max-doc-length = 120` but then I found this: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#syntax-and-organization
Do you think it is worth it?

Reply:
I'll leave it to you if you don't want to make that change, but note that they mention keeping it to 79 to make it more readable, and in this case, I think it makes it less readable.

If you end up keeping the wrapping, please add additional spaces to the wrapped line. For example:
```
# .. event_description: emitted when the user registration process in the LMS
#    is completed.
```
Thank you.

Reply:
Hello @robrap, we moved forward with your recommendation! Thank you so much. Also, we started working on the steps described here: https://code-annotations.readthedocs.io/en/latest/contrib/how_to/add_new_annotations_and_extracted_docs.html

Reply:
Great. Thank you. I hope you find it useful. One warning, that maybe we should add to the doc, is that we don't yet have a solution for pulling these annotations from dependencies. I'm not sure when and if this would apply to these annotations in particular, but something to keep in mind. It is a future enhancement that could be added later.

Reply:
Thanks for the heads up :+1: we'll look into